### DISCUSSION TITLE
Single shared space possible?

### DISCUSSION THREAD

**Discussion by becks0815** [TandoorRecipes/recipes#2278] [Q&A]
Just installed tandoor and enabled the possibility to create additional users. 

My question now is:

As far as I understand each user has his/her own space for receipts, and at least by default I can't see the receipis others have added to their space. Is there an option to either make them visible or to create a shared space for receipts to make this happen?

**Comment by smilerz** [comment]
What do you mean by "enabled the possibility to create additional users"?

The owner of a space can invite others on the `Space Settings` page (top right menu).

**Comment by becks0815** [comment]
Each user gets his own space, and other users can't see the content - at least on my installation. What I would like to achieve is that each user has his own space to create/modify/delete receipts, and other users can view and search for them on the web page. This results in a web page with 2 or more users, everyone can manage his/her receipts and everyone can see/read/search what others have created/saved.

**Reply by smilerz** [reply]
Spaces are not intended to work that way - if all content should be visible to all users they need to be in the same space.

**Comment by becks0815** [comment]
I understand that this approach makes sense in places where I want to keep my data separated and secret from others, like my banking details or my private emails, but if I look at public common sites for cooking recipes like chefkoch.de, they share the same approach I am looking for: a user can register, create/upload receipts, modify them, and they are automatically shared so others can see and use them.

What makes tandoor recipes so special that I have to manually grant access to my data and they are not shared to the public (aka my wife in my case) by default?

**Reply by smilerz** [reply]
You can have multiple users in a space - then everyone sees all of the recipes.  Or you can create separate spaces that are private and not shared. What reason do you have for not having your wife in the space you created for yourself?

**Reply by becks0815** [reply]
I will give it a try then, thx.
